externs/browser/ie_dom.js

```diff
@@ -1025,7 +1025,7 @@ Element.prototype.componentFromPoint = function(iCoordX, iCoordY) {};
 
 
 /**
- * @type {boolean}
+ * @type {string}
```

Comment:
This is one of those unfortunate cases where we can't fix the type easily because too much Google-internal code has come to depend on the current definition.

The best that can be done at the moment is to make this a `string|boolean`. Then, in a future where all internal code has been fixed to use strings, we can tighten the definition.

Reply:
*sigh* I was worried that would be the case, but hoped it would be treated as a bugfix and so the code would be fixed instead of definitions.

I can change it to the broader type.

Reply:
Done.

> Then, in a future where all internal code has been fixed to use strings, we can tighten the definition.

Is this tracked by the internal bug you mentioned? As in, do you think there's a chance of this being removed someday?